lms/djangoapps/courseware/views.py

```diff
@@ -982,7 +982,7 @@ def _progress(request, course_key, student_id):
             if certs_api.get_active_web_certificate(course) is not None:
```

Comment:
Looks like we can remove the certs_api.has_html_certificates_enabled check above now, no?